environment.root.yml

```diff
@@ -1,15 +1,8 @@
 name: mirflowz
 channels:
-  - bioconda
   - conda-forge
-  - defaults
+  - bioconda
 dependencies:
-  - graphviz=2.40.1
-  - jinja2=2.11.2
-  - networkx=2.5
-  - pygments=2.6.1
-  - python=3.7.4
+  - python=3.9.16
   - singularity=3.5.2
-  - snakemake=6.10.0
-  - unzip=6.0
-  - wget==1.20.1
+  - snakemake=7.24.0
```

Comment:
Remove Snakemake and Python from this one, along with the `conda-forge` channel.